Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "portus"
-version = "0.2.3"
-authors = ["Akshay Narayan <akshayn@csail.mit.edu>", "Frank Cangialosi <frankc@csail.mit.edu", "Deepti Raghavan <deeptir@stanford.edu"]
+version = "0.2.4"
```

Comment:
`0.3.0`: because it's a breaking change, we want to make sure downstream crates which write `portus = "0.2"` as their SemVer dependency don't pull in this change and break.